lib/services/internalEngine/index.js

```diff
@@ -571,6 +570,41 @@ class InternalEngine {
     })
       .catch(error => Bluebird.reject(this.esWrapper.formatESError(error)));
   }
+
+  /**
+   * Apply default mapping to the collection but preserve existing kuzzle metadata
+   *
+   * @param {string} index
+   * @param {string} collection
+   * @returns {Promise.<object>} returns the updated default mapping
+   */
+  applyDefaultMapping (index, collection, mapping) {
+    const defaultMapping = mapping;
+
+    return this.getMapping({ index, type: collection }, true)
+      .then(mappings => {
+        const existingMapping = mappings[index].mappings[collection];
+
+        // Preserve old version of kuzzle metadata mapping
+        if (existingMapping.properties && existingMapping.properties._kuzzle_info) {
+          Object.assign(
+            defaultMapping._kuzzle_info.properties,
```

Comment:
I checked, and the `mapping` parameter comes from the cache (index cache or configuration data).
You should not mutate it and, therefore, you have to change this Object.assign call